README.md

```diff
@@ -15,251 +15,191 @@
 
-```java
-public void configureMemoryUsage(JDABuilder builder) {
-    // Disable cache for member activities (streaming/games/spotify)
-    builder.disableCache(CacheFlag.ACTIVITY);
+We recommend reading the guide on [caching and intents](https://jda.wiki/using-jda/gateway-intents-and-member-cache-policy/) to get a feel for configuring your bot properly. Here are some possible use-cases:
 
-    // Only cache members who are either in a voice channel or owner of the guild
-    builder.setMemberCachePolicy(MemberCachePolicy.VOICE.or(MemberCachePolicy.OWNER));
+### Example: Message Logging
 
-    // Disable member chunking on startup
-    builder.setChunkingFilter(ChunkingFilter.NONE);
+Simply logging messages to the console. Making use of [JDABuilder][JDABuilder], the intended entry point for smaller bots that don't intend to grow to thousands of guilds.
 
-    // Disable presence updates and typing events
-    builder.disableIntents(GatewayIntent.GUILD_PRESENCE, GatewayIntent.GUILD_MESSAGE_TYPING);
+Starting your bot and attaching an event listener, using the right [intents][GatewayIntent]:
 
-    // Consider guilds with more than 50 members as "large". 
-    // Large guilds will only provide online members in their setup and thus reduce bandwidth if chunking is disabled.
-    builder.setLargeThreshold(50);
+```java
+public static void main(String[] args) {
+  JDABuilder.createLight(token, EnumSet.of(GatewayIntent.GUILD_MESSAGES, GatewayIntent.MESSAGE_CONTENT))
+      .addEventListener(new MessageReceiveListener())
+      .build();
 }
 ```
 
-### Listening for Events
+> [!CAUTION]
+> This also makes use of a **privileged** intent, which you have to explicitly enable in your application dashboard when you create your bot.
 
-The event system in JDA is configured through a hierarchy of classes/interfaces.
-We offer two implementations for the `IEventManager`:
+Your event listener could look like this:
 
-- **InterfacedEventManager** which uses an `EventListener` interface and the `ListenerAdapter` abstract class
-- **AnnotatedEventManager** which uses the `@SubscribeEvent` annotation which can be applied to methods
+```java
+public class MessageReceiveListener extends ListenerAdapter {
+  @Override
+  public void onMessageReceived(MessageReceivedEvent event) {
+    System.out.printf("[%s] %#s: %s\n",
+      event.getChannel(),
+      event.getAuthor(),
+      event.getMessage().getContentDisplay());
+  }
+}
+```
 
-By default, the **InterfacedEventManager** is used.
-Since you can create your own implementation of `IEventManager`, this is a very versatile and configurable system.
-If the aforementioned implementations don't suit your use-case you can simply create a custom implementation and
-configure it on the `JDABuilder` with `setEventManager(...)`.
+You can find a more thorough example with the [MessageLoggerExample](https://github.com/discord-jda/JDA/blob/master/src/examples/java/MessageLoggerExample.java) class.
```

Comment:
Perhaps we should also added:
"You can also find the full list of events [here], along with the methods provided by ListenerAdapter [here]